logger/init_test.go

```diff
@@ -51,3 +51,47 @@ func TestFormatterWithForcedPrefixFileOutput(t *testing.T) {
 		t.Error("Error removing test logs file:" + err.Error())
 	}
 }
+
+func Test_GetLooger(t *testing.T) {
+	tests := []struct {
+		name          string
+		env           string
+		expectedLevel logrus.Level
+	}{
+		{
+			name:          "default",
+			env:           "",
+			expectedLevel: logrus.InfoLevel,
+		},
+		{
+			name:          "error",
+			env:           "error",
+			expectedLevel: logrus.ErrorLevel,
+		},
+		{
+			name:          "warn",
+			env:           "warn",
+			expectedLevel: logrus.WarnLevel,
+		},
+		{
+			name:          "info",
+			env:           "info",
+			expectedLevel: logrus.InfoLevel,
+		},
+		{
+			name:          "debug",
+			env:           "debug",
+			expectedLevel: logrus.DebugLevel,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			os.Setenv("TYK_LOGLEVEL", tt.env)
```

Comment:
@mativm02 missed the defer of unset of the env after the test

Reply:
@tbuchaillot I will add it! thanks